cves/kernel/CVE-2016-6130.yml

```diff
@@ -369,9 +365,9 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-lessons:
+  answer: true
+  note:  In a way, a time of check, time of use vulnerability is changing the order of operations. But that can be seen as a stretch to some.
```

Comment:
Talk about the solution for this specific vulnerability. I noticed that the fix did include changing the order of operations.

Reply:
I went through and changed it to talk about the fix itself. I must've misread or misunderstood the question in my first draft.